devices/Utilities/Utils.hpp

```diff
@@ -149,5 +149,21 @@ class iDynTreeHelper::State::integrator
     double saturate(double val, double lowerLimit, double upperLimit);
 };
 
+// Yarp
+#include <yarp/sig/Vector.h>
+
+class DataBuffersConversionHelper {
+
+public:
+
+    DataBuffersConversionHelper();
+    ~DataBuffersConversionHelper();
+
+    void setBufferVecSize(yarp::sig::Vector&, size_t&);
+    void setBuffer(yarp::sig::Vector&, std::vector<double>&);
+    void setBuffer(yarp::os::Bottle&, std::vector<std::string>&);
```

Comment:
These are basically static methods since the do not change the state of the class. Maybe you can transform them to regular functions? If you want / need a scope, maybe you can add them in a dedicated namespace? Take the Eigen helpers of iDynTree as reference.

Reply:
Ref: https://github.com/robotology/idyntree/blob/master/src/core/include/iDynTree/Core/EigenHelpers.h